cookbook/implement-entity-extensible.rst

```diff
@@ -0,0 +1,163 @@
+How to implement an entity extensible?
+======================================
+
+Sulu uses the PersistenceBundle to provide an easy way to replace or extend internal entities.
```

Comment:
Would omit the `internal`, since this feature can also be used with other bundles.